pymemcache/client/base.py

```diff
@@ -82,10 +82,11 @@ def _parse_hex(value):
 
 def _check_key(key, key_prefix=b''):
     """Checks key and add key_prefix."""
-    if isinstance(key, six.text_type):
+    allowed_str_types = (six.text_type, six.string_types)
```

Comment:
This could be global so we don't need to rebuilt the tuple each time this function runs.